packages/openneuro-components/src/mock-content/facet-content.jsx

```diff
@@ -70,22 +76,134 @@ export const modalities = [
   },
 ]
 
-export const showDatasetsRadio = [
+export const show_available = [
+  { label: 'All', value: 'all' },
+  { label: 'Following', value: 'following' },
+  { label: 'My Uploads', value: 'my_uploads' },
+  { label: 'My Bookmarks', value: 'bookmarked' },
+]
+
+export const showMyUploads_available = [
+  { label: 'Public', value: 'public' },
+  { label: 'Shared with Me', value: 'shared_with_me' },
+  { label: 'Invalid', value: 'invalid' },
+]
+
+export const diagnosis = [
+  {
+    label: "Alzheimer's",
+    value: 'alzheimers',
+    children: null,
```

Comment:
This shouldn't be necessary. You might be getting a type error because FacetSelect.props.selected.children is not optional.
```suggestion
```